js/modules/loki_message_api.js

```diff
@@ -31,150 +30,154 @@ const filterIncomingMessages = async messages => {
   return newMessages;
 };
 
+const calcNonce = async (messageEventData, pubKey, data64, timestamp, ttl) => {
```

Comment:
do you need the `async` here as it doesn't call `await` ?

Reply:
Tru